README.md

```diff
@@ -27,98 +25,98 @@ Inside each `.json` file is a single array containing objects. It's easiest to
 illustrate the structure of these with an example:
 
 ```json
-    {
-        "description": "the description of the test case",
-        "schema": {"the schema that should" : "be validated against"},
-        "tests": [
-            {
-                "description": "a specific test of a valid instance",
-                "data": "the instance",
-                "valid": true
-            },
-            {
-                "description": "another specific test this time, invalid",
-                "data": 15,
-                "valid": false
-            }
-        ]
-    }
+{
+    "description": "The description of the test case",
+    "schema": {
+        "description": "The schema against which the data in each test is validated",
+        "type": "string"
+    },
+    "tests": [
+        {
+            "description": "Test for a valid instance",
+            "data": "the instance to validate",
+            "valid": true
+        },
+        {
+            "description": "Test for an invalid instance",
+            "data": 15,
+            "valid": false
+        }
+    ]
+}
 ```
 
-In short: a description, a schema under test, and some tests, where each
-test in the `tests` array is an objects with a description of the case
-itself, the instance under test, and a boolean indicating whether it
-should be valid or invalid.
+In short: a description, a schema under test, and some tests, where each test
+in the `tests` array is an objects with a description of the case itself, the
+instance under test, and a boolean indicating whether it should be valid
+or invalid.
 
-Test Subdirectories
--------------------
+## Test Subdirectories
 
 There is currently only one subdirectory that may exist within each draft
 directory. This is:
 
 1. `optional/`: Contains tests that are considered optional.
 
-Coverage
---------
+## Coverage
 
-Drafts 07, 06, 04 and 03 should have full coverage, with drafts 06 and
-07 being considered current and actively supported.
+Drafts 07, 06, 04 and 03 should have full coverage, with drafts 06 and 07 being
+considered current and actively supported.
 
-Draft 2019-09 support is under development. Contributions are very
-welcome, especially from implementers as they add support to their own
-implementations.
+Draft 2019-09 support is under development. Contributions are very welcome,
+especially from implementers as they add support to their own implementations.
 
-If you see anything missing from the current supported drafts, or incorrect
-on any draft still accepting bug fixes, please file an issue or submit a PR.
+If you see anything missing from the current supported drafts, or incorrect on
```

Comment:
I think we should drop this line altogether -- it's not very clear whether it's asking for PRs to the spec, or to the test suite.  PRs to the spec aren't being accepted to any but the latest, and PRs to the test suite are always welcome as long as the draft has coverage at all (I think someone proposed dropping draft3 entirely, but I can't find that GHI at the moment).

Reply:
I'd prefer to leave for a future, more substantial PR.